package-apisix.sh

```diff
@@ -3,13 +3,25 @@ set -euo pipefail
 set -x
 mkdir /output
 dist=$(cat /tmp/dist)
+
+# Determine the dependencies
+dep_pcre="pcre"
+dep_libpcre="pcre-devel"
+if [ "$PACKAGE_TYPE" == "deb" ]
+then
+	dep_pcre="libpcre3"
+	dep_libpcre="libpcre3-dev"
```

Comment:
The dev version of the package is only required during building.

Reply:
@spacewander Done. Thanks.